packages/swingset-theme-hashicorp/src/components/text/body.tsx

```diff
@@ -0,0 +1,15 @@
+import { cva, VariantProps, cx } from 'class-variance-authority'
+import { type HTMLAttributes } from 'react'
+
+type HeadingProps = HTMLAttributes<HTMLHeadingElement>
```

Comment:
`HTMLHeadingElement` is for `h1, h2, h3, etc`, but these props are being passed to a `HTMLParagraphElement`.